api/HTMLElement.json

```diff
@@ -570,14 +570,62 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": null
+              "version_added": "45"
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options": {
```

Comment:
Maybe give this a more specific name, given the description? Maybe `preventScroll_option`? I couldn't find another example of this sort of thing in the data, so I don't think we have a convention to adhere to.

Reply:
Looking at this with fresh eyes, I realized this is similar to init objects on constructors. I now think that neither is the right approach. We probably need a structure with a generic options/init and its actual options as children. This would give us the flexibility to programmatically how this info is displayed. For example, we could make the list of options expandable/collapsable.

@Elchi3 @chrisdavidmills

Reply:
preventScroll_option would be consistent with what we have elsewhere so far. C.f.:

- https://github.com/mdn/browser-compat-data/blob/f4f8ce8eb117281c5037d15cac257293fd19301c/api/AudioContext.json#L152
- https://github.com/mdn/browser-compat-data/blob/f4f8ce8eb117281c5037d15cac257293fd19301c/api/Clients.json#L224

I agree it is not perfect, but it works for now. I think coming up with additional structure should be discussed separately.